doc/source/getting_started.rst

```diff
@@ -27,6 +29,28 @@ You will need the following:
    key is required due to an `upstream bug in
    paramiko <https://github.com/paramiko/paramiko/issues/2191>`_)
 
+Required dependencies for smooth install
+----------------------------------------
+
+On a Debian 12 cloud server
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
this sounds like being about the target server, but it's really the client side, right?